pkg/network/state.go

```diff
@@ -287,7 +287,7 @@ func (ns *networkState) RegisterClient(id string) {
 func getConnsByKey(conns []ConnectionStats, buf []byte) map[string]*ConnectionStats {
 	connsByKey := make(map[string]*ConnectionStats, len(conns))
 	for i, c := range conns {
-		key := c.ByteKey(buf)
+		key := c.ByteKeyNAT(buf)
```

Comment:
How does this handle the scenario where we don't have the NAT translation yet?

Reply:
If we don't have NAT info, we would potentially mismatch some connections, but this is just part and parcel of how we attach NAT info. The inverse where we do not use NAT info here would result in mismatches too and in some environments (kube-proxy, etc.) this leads to a lot of underflows because of mismatches. 

This is consistent with the backend and UI.